src/Pokeview/Pokeview.scss

```diff
@@ -0,0 +1,36 @@
+@import '../styles/_GrayBubble.scss';
+
+.j-pokeview {
+
+    &-welcome {
+        @include horizontal-bar-right;
+    }
+
+    &-container > img {
+        width: 35%;
+        position: absolute;
+        left: 6%;
+        bottom: 13%;
+
+        @media screen and (max-width: 773px) {
+            width: 30%;
+            left: 10%;
+        }
+
+    }
+
+    &-container > div {
```

Comment:
I encourage you to use a class instead of a simple div selector

Reply:
Fixed